functions/routes/authentication.js

```diff
@@ -0,0 +1,44 @@
+var express=require("express");
+var router=express.Router();
+
+// a route which we will be using to authenticate API calls for NIN pending our approval, it requires a post request to be activated
+router.post("/",(request,response)=>{
+    // extract the phone number and NIN from the body of the response
+    const {nin,phoneNumber}=request.body;
```

Comment:
what is `nin`?

Reply:
National identification number, there is a government API which can verify the NINs and phone numbers attaxhed

Reply:
It's a unique number given as our national identity.

Reply:
okay, fix all the linting errors then you are all set to merge this code down